e2e/support/utils/on-wrapper.ts

```diff
@@ -0,0 +1,52 @@
+import _ from "underscore";
+
+export class OnWrapper {
```

Comment:
We need it, see https://github.com/haim-io/cypress-image-diff/issues/183#issuecomment-1749005435